ios/Podfile

```diff
@@ -17,9 +18,9 @@ target 'ExpensifyCash' do
   #
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable these next few lines.
-  use_flipper!
+  # use_flipper!
   post_install do |installer|
-    flipper_post_install(installer)
+    # flipper_post_install(installer)
```

Comment:
Oops forgot to remove this guy. It needed so that iOS will build on dev for now. 🙃